def test_fail():
    assert (2+1)==4
    assert (2+1)==4